certified-connectors/Viafirma/apiDefinition.swagger.json

```diff
@@ -0,0 +1,341 @@
+                  "x-ms-dynamic-values": {
+                    "operationId": "WhoIAm",
+                    "value-path": "code",
+                    "value-collection": "groupsInfo",
+                    "value-title": "code"
+                  }
+                },
+                "workflow": {
+                  "type": "object",
+                  "properties": {
+                    "type": {
+                      "type": "string",
+                      "description": "type",
+                      "title": "",
+                      "default": "WEB",
+                      "x-ms-visibility": "internal"
+                    }
+                  },
+                  "description": "workflow",
+                  "required": [
+                    "type"
+                  ]
+                },
+                "notification": {
+                  "type": "object",
+                  "properties": {
+                    "text": {
+                      "type": "string",
+                      "description": "text"
+                    },
+                    "detail": {
+                      "type": "string",
+                      "description": "detail"
+                    },
+                    "sharedLink": {
+                      "type": "object",
+                      "properties": {
+                        "email": {
+                          "type": "string",
+                          "description": "email",
+                          "title": ""
+                        },
+                        "subject": {
+                          "type": "string",
+                          "description": "subject"
+                        }
+                      },
+                      "description": "sharedLink",
+                      "required": [
+                        "email"
+                      ]
+                    }
+                  },
+                  "description": "notification",
+                  "required": [
+                    "sharedLink"
+                  ]
+                },
+                "document": {
+                  "type": "object",
+                  "properties": {
+                    "templateCode": {
+                      "type": "string",
+                      "description": "templateCode",
+                      "title": ""
+                    }
+                  },
+                  "description": "document",
+                  "required": [
+                    "templateCode"
+                  ]
+                },
+                "callbackMails": {
+                  "type": "string",
+                  "description": "callbackMails"
+                }
+              },
+              "default": {
+                "groupCode": "your_group_here",
+                "workflow": {
+                  "type": "WEB"
+                },
+                "notification": {
+                  "text": "TEST Firma web",
+                  "detail": "notificado v\u00eda email",
```

Comment:
@anfuca Unicodes - The connector's swagger still have unicodes which may not be supported in all the Power Automate page, Can you please replace all the unicodes with respective characters in the English language.